integration-test/2022-disputed-borders.py

```diff
@@ -84,6 +85,7 @@ def test_admin_level_3_state(self):
                 'kind': 'disputed_reference_line',
                 'kind:xx': 'country',
                 'disputed': True,
+                'kind:yy': 'country',
```

Comment:
Please extend this test so there is a **kind:zz:** `unrecognized_disputed_reference_line` in the test here.